repository/release.go

```diff
@@ -63,6 +69,50 @@ func (r *ReleaseRepository) Read(ctx context.Context, projectID, releaseID uuid.
 	return model.ToSvcRelease(rls), nil
 }
 
+func (r *ReleaseRepository) Update(
+	ctx context.Context,
+	projectID,
+	releaseID uuid.UUID,
+	fn svcmodel.UpdateReleaseFunc,
+) (rls svcmodel.Release, err error) {
+	tx, err := r.dbpool.Begin(ctx)
+	if err != nil {
+		return svcmodel.Release{}, fmt.Errorf("failed to start transaction: %w", err)
+	}
+	defer func() {
+		err = util.FinishTransaction(ctx, tx, err)
+	}()
+
+	// Project ID is not needed in the query because releaseID is primary key
+	// But it is added for security reasons
+	// To make sure that the release belongs to the project that is passed from the service
```

Comment:
In that case, the read/patch endpoint would have to include the release title instead of the ID:

/projects/{project_id}/releases/{release_title}

Otherwise, I would not be able to fetch the release.

I will stick with the current implementation for now. However, feel free to comment if I misunderstood your suggestion. Thank you

Reply:
You are right. It might trigger too many unnecessary changes and inconsistencies.